### PR TITLE
[PNA] Add IP address space definition and update rollout timeline

### PR DESCRIPTION
**Pull request by iVanlIsh**
- Add IP address space definition information into the blog.

- PNA warning preflights are rolling out in Chrome 104 once more. Enforcement will be enabled in 107 at the earliest.

https://deploy-preview-3144--developer-chrome-com.netlify.app/blog/private-network-access-preflight/

### CODE REVIEW

**Review by agektmr — APPROVED** [review]
LGTM

**Comment by agektmr** [comment]
@heyawhite @mihajlija Could either of you review this pull request and merge please?

File: site/en/blog/private-network-access-preflight/index.md [hunk 1]
```diff
@@ -93,6 +95,28 @@ a request from a public website (`https://example.com`) to a private website
 (`http://router.local`), or a request from a private website to localhost.
 {% endAside %}
 
+### How do PNA classify IP addresses and identify private network
```
Comment (mihajlija) [inline site/en/blog/private-network-access-preflight/index.md:98]:
```suggestion
### How does PNA classify IP addresses and identify a private network 
```

File: site/en/blog/private-network-access-preflight/index.md [hunk 2]
```diff
@@ -93,6 +95,28 @@ a request from a public website (`https://example.com`) to a private website
 (`http://router.local`), or a request from a private website to localhost.
 {% endAside %}
 
+### How do PNA classify IP addresses and identify private network 
+
+The IP addresses are classified into three IP address spaces: `public`,
+`private` and `local`.
```
Comment (mihajlija) [inline site/en/blog/private-network-access-preflight/index.md:101]:
```suggestion
The IP addresses are classified into three IP address spaces: 
- `public`
- `private`
- `local`
```

File: site/en/blog/private-network-access-preflight/index.md [hunk 3]
```diff
@@ -93,6 +95,28 @@ a request from a public website (`https://example.com`) to a private website
...
+
+**A local IP address space** contains IP addresses which is either an IPv4
+loopback address (`127.0.0.0/8`) defined in section 3.2.1.3 of [RFC1122](https://tools.ietf.org/html/rfc1122)
+or an IPv6 loopback address (`::1/128`) defined in section 2.5.3 of [RFC4291](https://tools.ietf.org/html/rfc4291).
```
Comment (mihajlija) [inline site/en/blog/private-network-access-preflight/index.md:105]:
```suggestion
**Local IP address space** contains IP addresses that are either IPv4
loopback addresses (`127.0.0.0/8`) defined in section 3.2.1.3 of [RFC1122](https://tools.ietf.org/html/rfc1122)
or IPv6 loopback addresses (`::1/128`) defined in section 2.5.3 of [RFC4291](https://tools.ietf.org/html/rfc4291).
```

File: site/en/blog/private-network-access-preflight/index.md [hunk 4]
```diff
@@ -93,6 +95,28 @@ a request from a public website (`https://example.com`) to a private website
...
+
+**A private IP address space** contains IP addresses that have meaning only
+within the current network, including `10.0.0.0/8`, `172.16.0.0/12` and
+`192.168.0.0/16` defined in [RFC1918](https://tools.ietf.org/html/rfc1918),
+link-local addresses `169.254.0.0/16` defined in [RFC3927](https://tools.ietf.org/html/rfc3927),
+unique local IPv6 unicast addresses `fc00::/7` defined in [RFC4193](https://datatracker.ietf.org/doc/html/rfc4193),
+link-local IPv6 unicast addresses `fe80::/10` defined in section 2.5.6 of [RFC4291](https://tools.ietf.org/html/rfc4291)
+and IPv4-mapped IPv6 addresses where the mapped IPv4 address is itself private.
```
Comment (mihajlija) [inline site/en/blog/private-network-access-preflight/index.md:113]:
```suggestion
**Private IP address space** contains IP addresses that have meaning only
within the current network, including `10.0.0.0/8`, `172.16.0.0/12` and
`192.168.0.0/16` defined in [RFC1918](https://tools.ietf.org/html/rfc1918),
link-local addresses `169.254.0.0/16` defined in [RFC3927](https://tools.ietf.org/html/rfc3927),
unique local IPv6 unicast addresses `fc00::/7` defined in [RFC4193](https://datatracker.ietf.org/doc/html/rfc4193),
link-local IPv6 unicast addresses `fe80::/10` defined in section 2.5.6 of [RFC4291](https://tools.ietf.org/html/rfc4291)
and IPv4-mapped IPv6 addresses where the mapped IPv4 address is itself private.
```

File: site/en/blog/private-network-access-preflight/index.md [hunk 5]
```diff
@@ -93,6 +95,28 @@ a request from a public website (`https://example.com`) to a private website
...
+
+**A public IP Address space** contains all the other addresses not mentioned previously.
```
Comment (mihajlija) [inline site/en/blog/private-network-access-preflight/index.md:115]:
```suggestion
**Public IP Address space** contains all other addresses not mentioned previously.
```

**Review by mihajlija — APPROVED** [review]

**Pull request merged**